go/src/koding/kites/kloud/provider/aws/aws_test.go

```diff
@@ -0,0 +1,11 @@
+package aws_test
+
+import (
+	"koding/kites/kloud/provider/aws"
+	"koding/kites/kloud/stack"
+)
+
+var (
+	_ stack.Machiner = (*aws.Machine)(nil)
+	_ stack.Stacker  = (*aws.Stack)(nil)
```

Comment:
Why are these checks not in `machine.go`/`stack.go` files? This is inconsistent with`stack.Validator` interface checks in `aws.go`.